src/test/resources/rca/rca_elected_master.conf

```diff
@@ -73,6 +73,12 @@
       "cpu-utilization-cluster-percentage" : 0.3,
       "io-total-throughput-cluster-percentage" : 0.3,
       "io-total-syscallrate-cluster-percentage" : 0.3
+    },
+    "cache-config": {
```

Comment:
Let us rename this "cache-config" to the actual name of your RCA. ideally we should keep the 1:1 mapping between the config and the RCA it belongs to to avoid confusion.

Reply:
Got it. Modified